serverless.yml

```diff
@@ -19,7 +19,7 @@ plugins:
 
```

Comment:
regarding plugin - serverless-single-page-app-plugin, I didn't see it in your dependencies

Reply:
I just made the changes. Thanks